### PR TITLE
fix: now submit doesn't require to select validDate

### PR DESCRIPTION
**Pull request by PC-11-00**
## Description
Submit button on edit standing instructions was not active until Validity to date is selected
## Related issues and discussion
#1981 

## Screenshots, if any

https://github.com/openMF/web-app/assets/76156941/511c318b-d376-45d3-b558-43944d5e9cdc

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/account-transfers/edit-standing-instructions/edit-standing-instructions.component.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export class EditStandingInstructionsComponent implements OnInit {
   /** Minimum date allowed. */
   minDate = new Date(2000, 0, 1);
   /** Maximum date allowed. */
-  maxDate = new Date();
+  maxDate = new Date(2100,0,1);
```
Comment (adamsaghy) [inline src/app/account-transfers/edit-standing-instructions/edit-standing-instructions.component.ts:42]: Do we need max date?

Comment (PC-11-00) [reply src/app/account-transfers/edit-standing-instructions/edit-standing-instructions.component.ts:42]: Actually I don't know if we need or not, it was already in code and it was till business date and while creating standing instruction it was till this so I thought I should make it till this

Comment (PC-11-00) [reply src/app/account-transfers/edit-standing-instructions/edit-standing-instructions.component.ts:42]: @adamsaghy should I remove max-date, if don't require?

Comment (adamsaghy) [reply src/app/account-transfers/edit-standing-instructions/edit-standing-instructions.component.ts:42]: @bharathcgowda Do we need max date for standing instructions?

Comment (bharathcgowda) [reply src/app/account-transfers/edit-standing-instructions/edit-standing-instructions.component.ts:42]: @adamsaghy @PC-11-00  is this max-date same as the "valid to date" in the UI? then Yes it is required

**Pull request merged**